x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java

```diff
@@ -325,10 +467,11 @@ private String executePolicy(String policyId) {
         }
     }
 
-    public static void blockMasterFromFinalizingSnapshotOnIndexFile(final String repositoryName) {
-        for(RepositoriesService repositoriesService : internalCluster().getDataNodeInstances(RepositoriesService.class)) {
-            ((MockRepository)repositoriesService.repository(repositoryName)).setBlockOnWriteIndexFile(true);
-        }
+    public static String blockMasterFromFinalizingSnapshotOnIndexFile(final String repositoryName) {
+        final String masterName = internalCluster().getMasterName();
+        ((MockRepository)internalCluster().getInstance(RepositoriesService.class, masterName)
+            .repository(repositoryName)).setBlockOnWriteIndexFile(true);
+        return masterName;
```

Comment:
@dakrone I had to make this change as the test I added uses the return value. The new implementation of this method is the same as it is in `master`, in `7.x` the implementation of this method was identical to `blockAllDataNodes` - I'm not sure exactly what was up with that. Just a heads up.

Reply:
Thanks for the heads up :+1: